website/src/content/api-reference/commercetools-frontend-application-components.mdx

```diff
@@ -597,6 +598,44 @@ const AccountPage = () => {
   />
   ```
 
+## TabHeader
```

Comment:
I'm not sure I would document this separately. It should be enough for now to document it within `TabularModalPage` component context.